### PR TITLE
added in step by step of local editing process

### PR DESCRIPTION
**Pull request by writerly03**
Hope these steps are clear enough; this is the process if using GH Desktop so not sure if more explanation is required if not using this interface

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,17 @@ make check
 # Run slower (to avoid failing links from servers throttling you)
 make check-cautious
 ```
+Once you have installed the above tools, these are the steps required to make edits to existing Handbook pages:
+
+1. Check that your local version of the main (master) is up to date
```
Comment (uditvira) [inline README.md:65]:
```suggestion
1. Check that your local version of the main (master) branch is up to date. If not, you should pull recent changes from GitHub. On the command line you'll have to first use `git checkout master` and then `git pull`.
```

File: README.md [hunk 2]
```diff
@@ -60,6 +60,17 @@ make check
...
+2. Create a new branch: select New branch (if using GitHub Desktop) and give it a title of whatever you're planning to work on e.g. roles-page-update
```
Comment (uditvira) [inline README.md:66]:
```suggestion
2. Create a new branch: select New branch (if using GitHub Desktop) and give it a title of whatever you're planning to work on e.g., roles-page-update. On the command line you can use `git checkout -b your-branch-name`.
```

File: README.md [hunk 3]
```diff
@@ -60,6 +60,17 @@ make check
...
+3. Make your edits or changes using a text editor
+4. Write a summary of the changes
```
Comment (uditvira) [inline README.md:68]:
```suggestion
4. If you're using the command line, you'll have to stage changes you want to commit. You can use `git add file-name` to do that.
```

File: README.md [hunk 4]
```diff
@@ -60,6 +60,17 @@ make check
 # Run slower (to avoid failing links from servers throttling you)
 make check-cautious
 ```
+Once you have installed the above tools, these are the steps required to make edits to existing Handbook pages:
+
+1. Check that your local version of the main (master) is up to date
+2. Create a new branch: select New branch (if using GitHub Desktop) and give it a title of whatever you're planning to work on e.g. roles-page-update
+3. Make your edits or changes using a text editor
+4. Write a summary of the changes 
+5. Commit the change
```
Comment (uditvira) [inline README.md:69]:
```suggestion
5. Commit the change and write a summary message for that commit. If using the command line, you can use `git commit -m "your summary message"
```

File: README.md [hunk 5]
```diff
@@ -60,6 +60,17 @@ make check
...
+6. Make a pull request to the main
```
Comment (uditvira) [inline README.md:70]:
```suggestion
6. Push the changes upstream to GitHub. With the command line you can use `git push`
7. Make a pull request to the master branch from your branch
```

File: README.md [hunk 6]
```diff
@@ -60,6 +60,17 @@ make check
...
+7. Assign a co-worker to review the changes
```
Comment (uditvira) [inline README.md:71]: If you accept my other suggestions you'll have to change the numbering
```suggestion
8. Assign a co-worker to review the changes
```

File: README.md [hunk 7]
```diff
@@ -60,6 +60,17 @@ make check
...
+8. Co-worker reviews and then accepts the pull request, and the update is made
```
Comment (uditvira) [inline README.md:72]: ibid
```suggestion
9. Co-worker reviews and then accepts the pull request, and the update is made  
```

**Review by uditvira — CHANGES_REQUESTED** [review]
Added instructions for command line as well

**Review by tripledoublev — APPROVED** [review]

**Review by tripledoublev — APPROVED** [review]

**Pull request merged**